### DISCUSSION TITLE
set boundaries for swarmplot

### DISCUSSION THREAD

**Discussion by stevenlis** [mwaskom/seaborn#3151] [Q&A]
https://seaborn.pydata.org/generated/seaborn.swarmplot.html

The current swarmplot symmetrically distributes data points along a line.
Could this be a feature request to add a constraint to only allow data points spread on one side.

One example I found is here:

https://www.theguardian.com/news/ng-interactive/2018/apr/05/women-are-paid-less-than-men-heres-how-to-fix-it
![CS-KhQk6Liy](https://user-images.githubusercontent.com/33796896/203462484-b153b42d-e616-49d4-a09a-8d0cc23cade2.png)

**Comment by lokmac** [comment]
I'd love to know how this was actually formed. Very clever the way dots that would otherwise have overlapped, even not in the same "bin", are stacked
